gordo/reporters/mlflow.py

```diff
@@ -185,7 +185,66 @@ def epoch_now() -> int:
     return _datetime_to_ms_since_epoch(datetime.now(tz=UTC))
 
 
-def get_batch_kwargs(machine: Machine) -> dict:
+def split_log_item_batches(
+    metrics: List[Metric],
+    params: List[Param],
+    n_max_metrics: int = 200,
```

Comment:
My parranoia sais that we should set this to 199, and n_max_params to 99 :-p

Reply:
Are we not sending / caring about tags?

Reply:
I manual-integration tested it with those values and things worked, but it couldn't hurt.

Due to the AzureML limits, the tag limits wouldn't come into affect here, as we are only using one tag per run, and the AzureML each limits (I discovered through manual testing) are independent of each other, as opposed to the MLFlow total sum limit that we won't reach due to the Azure limits.

Reply:
Oh, and I there is a unit test to check the batching logic.